requirements/ftest.txt

```diff
@@ -1,7 +1,7 @@
 flask==2.3.2
 flask_limiter==3.3.1
 faker==18.11.2
-mysql-connector-python==8.1.0
+mysql-connector-python==9.0.0
```

Comment:
why? Major jumps scare me. :)

Reply:
Transitive dependency bump - `mysql-connector-python` needed protobuf of unsupported version:

```
mysql-connector-python==8.1.0
└── protobuf [required: >=4.21.1,<=4.21.12, installed: 4.21.12]
```

Elastic Python Agent repo has 5.27+ protobuf declared as a dependency.

This package is also only used by functional tests, so should not affect production code.